### ISSUE TITLE
make a route to promote your self as a web dev

### ISSUE THREAD

**Issue by GN101** [GN101/consumerism_app#51]
create a page where an potential employer can just see a quick summary of your practiced skills without having to view your source code (perhaps he is unimpressed? Bored? who knows? and who cares? Just spoon feed him what you want to convey).

This could be the `About` page, where you'll fill some personal info about you (including a link to your CV. Below that, perhaps have an area where you name everything you used to create this app (that is even a bit noteworthy - don't just spam useless shit), mention any tickets you created on your own (shows ability to think for yourself / ability to see ways to optimize the product on your own), list any optimizations you applied to the app (more on this once you finish all your current tickets).

In summary:
- [x] fill in the `About` page with your CV + a bit of info about you
- [x] a panel where you refer to all your noteworthy aspects of your work in this app
- [ ] perhaps mention what you plan to invest into the future (e.g. Node.js or anything else you have in mind)
